### DISCUSSION TITLE
We should provide support for the new Twitter logo X in the footer and metabox

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#59] [Enhancement Requests]
![image](https://github.com/amnestywebsite/humanity-theme/assets/37809410/5e4d14e9-4502-48c2-ac14-29f6a959a034)

![image](https://github.com/amnestywebsite/humanity-theme/assets/37809410/b4e2e2b9-a0d0-40d3-94f5-109c1a2378a0)

https://github.com/amnestywebsite/amnesty-wp-theme/discussions/2584

**Comment by daveamnesty** [comment]
@AmnestyAM FYI the outdated logo was raised internally at Amnesty by the Crisis team, via RSPU. I've edited the title of this discussion to reflect the fact that the outdated logo is also used on the metabox, which is the issue raised for amnesty.org. (as we've already updated the footer using FSE)
